components/helm-broker/internal/ybundle/automock/extended.go

```diff
@@ -22,33 +22,33 @@ func (m *Repository) ExpectErrorOnIndexReader(outErr error) {
 }
 
 func (m *Repository) ExpectOnBundleReader(givenName ybundle.BundleName, givenVersion ybundle.BundleVersion, bundleReader io.Reader) {
```

Comment:
You can change the package name from `ybundle` to `bundle`.

Reply:
changed